méli-mélo/2021-05-conjunction/a11y-1-conjunction.json

```
@@ -0,0 +1,651 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+    "@type": [
+        "earl:Assertion",
+        "acr:AssessmentReport"
+    ],
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/m%C3%A9li-m%C3%A9lo-demos/2024-09-kejimkujik/2021-05-conjunction",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Assess the GCWeb meli-melo conjunction-and_or page according to Standard on Web Accessibility (SWA)",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "conjunction-and_or"
+        }
+    },
+    "earl:assertedBy": {
+        "foaf:name": "Service Canada - Principal Publisher",
+        "foaf:homepage": "https://github.com/ServiceCanada",
+        "@type": [
+            "earl:Assertor",
+            "foaf:Organization"
+        ],
+        "earl:mainAssertor": {
+            "foaf:name": "Oakes Christopher (Github: @Christopher-O)",
+            "foaf:homepage": "https://github.com/Christopher-O",
+            "@type": [
+                "earl:Assertor",
+                "foaf:Person"
+            ],
+            "foaf:member": {
+                "@id": "_:wave_browser_extension",
+                "doap:name": "WAVE web accessibility evaluation tool, browser extension",
+                "doap:homepage": "https://wave.webaim.org/extension/",
+                "@type": [
+                    "earl:Assertor",
+                    "earl:Software"
+                ]
+            }
+        }
+    },
+    "dct:date": "2024-06-11",
+    "dct:description": "Evaluation of the meli-melo conjunction-and_or page. Evaluated all Success Criteria at Level A and AA of WCAG 2.1",
+    "acr:standard": "https://www.w3.org/TR/WCAG21",
+    "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+    "earl:result": [
+        {
+            "@id": "_:sc_1.1.1",
+            "earl:test": "WCAG21:non-text-content(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
+        {
+            "@id": "_:sc_1.2.1",
+            "earl:test": "WCAG21:audio-only-and-video-only(Level A)",

```

Must match the ID in the W3C specification of WCAG 2.1

```suggestion
            "earl:test": "WCAG21:audio-only-and-video-only-prerecorded",
```